### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by nikoandpiko**
Fixes various grammar, typos, and updates YK image link to display properly.

Additionally cleaning up markdown syntax if desired.

### CODE REVIEW

File: README.md
Comment (dkocher) [inline README.md:82]: There is a new typo introduced.

File: README.md
Comment (dkocher) [inline README.md:41]: Why do you consider dashes deprecated syntax?

**Review by dkocher — CHANGES_REQUESTED** [review]

File: README.md
Comment (nikoandpiko) [reply README.md]: I was not familiar. It has now been reverted.

File: README.md
Comment (nikoandpiko) [reply README.md]: Following [this standard](https://github.com/DavidAnson/markdownlint/blob/v0.26.2/doc/Rules.md#md004), but it is only a suggestion. It's now been reverted.

**Review by dkocher — CHANGES_REQUESTED** [review]
Looks good to me. Please sqash the commits to merge.

**Comment by nikoandpiko** [comment]
@dkocher Any more changes requested? Merging is blocked until approved.

**Comment by dkocher** [comment]
> @dkocher Any more changes requested? Merging is blocked until approved.

Please sqash the commits.

**Comment by nikoandpiko** [comment]
Apologies. Need to fix this -- ran into issues with git.

**Review by dkocher — APPROVED** [review]

**Pull request merged**
